magmi/engines/magmi_productimportengine.php

```diff
@@ -469,19 +481,6 @@ public function initAttrInfos($cols)
     }
 
     /**
-     * Fetches the entity type for "catalog_product" and stores it to $this->prod_etype (if not already done).
-     */
-    private function fetchProdEType() {
-        if ($this->prod_etype == null)
-        {
-            // Find product entity type
-            $tname = $this->tablename("eav_entity_type");
-            $this->prod_etype = $this->selectone("SELECT entity_type_id FROM $tname WHERE entity_type_code=?",
-                                                 "catalog_product", "entity_type_id");
-        }
-    }
-
-    /**
```

Comment:
To group it with a similar method. Might prevent others from re-implementing it again.